src/core.ts

```diff
@@ -34,7 +34,7 @@ function generateResultsToUndo(
         if (!resultOfLint) throw new Error(`The result of ${resultToFix.filePath} is not found.`);
         // NOTE: THIS IS HACK.
         // Usually, the result with `output` property does not contain `source` property.
-        // However, `source` property is required for undoTransformation.
+        // However, `source` property is required for undo.
```

Comment:
I want it to be `to`!

```suggestion
        // However, `source` property is required to undo.
```